.dotnet/examples/Assistants/Example05_AssistantsWithVision.cs

```diff
@@ -17,7 +18,7 @@ public void Example05_AssistantsWithVision()
         AssistantClient assistantClient = openAIClient.GetAssistantClient();
 
         OpenAIFile pictureOfAppleFile = fileClient.UploadFile(
-            "picture-of-apple.jpg",
+            Path.Combine("Assets", "picture-of-apple.png"),
             FileUploadPurpose.Vision);
         Uri linkToPictureOfOrange = new("https://platform.openai.com/fictitious-files/picture-of-orange.png");
```

Comment:
We need to update this URL once the orange picture gets merged into the main OpenAI repo. The sample is green now but it only outputs the following text:
```
--- Run started! ---
```